src/main/java/clickup/ui/pages/SpacePage.java

```diff
@@ -50,7 +58,16 @@ public void addNewSpace(String nameSpace) {
      * @param nameSpace parameter string.
      * @return boolean result.
      */
-    public boolean isFoundNameSpace(String nameSpace) {
+    public boolean isFoundNameSpace(final String nameSpace) {
         return getDriver().getPageSource().contains(nameSpace);
     }
+
+    /**
+     * Method let log out of the page.
```

Comment:
Please change the documentation according to our Code Conventions.

Reply:
Done.